static/src/javascripts/projects/common/modules/component.js

```diff
@@ -107,7 +109,11 @@ define([
 
         this.elem = template;
         this._prerender();
-        bonzo(container)[this.manipulationType](this.elem);
+
+        fastdom.write(function () {
+            bonzo(container)[this.manipulationType](this.elem);
+        });
+
         this._ready();
```

Comment:
It appears someone [thought of this already](https://github.com/wilsonpage/fastdom#fastdomdeferframes-callback-context) will try it out.